components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApiProductsApiServiceImpl.java

```diff
@@ -852,7 +852,7 @@ private boolean isAuthorizationFailure(Exception e) {
      */
     private void validateAPIProduct(String apiProductId) throws APIManagementException {
         APIProvider apiProvider = RestApiCommonUtil.getLoggedInUserProvider();
-        APIInfo apiInfo = apiProvider.getAPIInfoByUUID(apiProductId);
+        APIInfo apiInfo = apiProvider.getAPIInfoByUUID(apiProductId, null);
```

Comment:
Shall we revert this back to use the default method?

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/13326/commits/6ddc11dad84c96692f671a8f12101549928ee28f